operator/cluster/clusterlogic.go

```diff
@@ -152,7 +153,7 @@ func AddCluster(clientset *kubernetes.Clientset, client *rest.RESTClient, cl *cr
 		log.Debugf("Custom postgres-ha configmap not found, creating configMap with default postgres-ha config file")
 		operator.AddDefaultPostgresHaConfigMap(clientset, cl, deploymentFields.IsInit, true, namespace)
 	} else {
-		configMap, found := kubeapi.GetConfigMap(clientset, config.GLOBAL_CUSTOM_CONFIGMAP, namespace)
+		configMap, found := kubeapi.GetConfigMap(clientset, strings.Trim(deploymentFields.ConfVolume, "\""), namespace)
```

Comment:
I'd recommend putting the Trimmed configmap in its own variable to make it easier to debug in the future.

Reply:
The trimmed string has now been separated into its own variable.